README.md

```diff
@@ -118,6 +114,15 @@ pip install auto-round-lib
 
 ## Model Quantization (CPU/Intel GPU/Gaudi/CUDA)
 
+### Environment Configuration 
+Here we list some of the most commonly used environment variables:
```

Comment:
no need to use a section, we need to keep the homepage as clean as possible.
Just claim how to use model scope is enough and move this **one sentence** to CLI usage